<launch>
  <!-- startup move_base node -->
  <!-- <include file="$(find roborts_nav)/launch/stdr_move_base.launch" /> -->

  <!-- startup amcl node -->
  <!-- <include file="$(find stdr_amcl)/launch/stdr_amcl.launch" /> -->

  <!-- startup patrol navigation node -->
  <node pkg="roborts_nav" type="patrol_nav.py" name="patrol_nav_node" output="screen" >
    <param name="rest_time"     value="0" />
    <param name="keep_patrol"   value="true" />
    <param name="random_patrol" value="false" />
    <param name="patrol_type"   value="0" />
    <param name="patrol_loop"   value="1" />
    <param name="patrol_time"   value="3" />
  </node>

</launch>
